debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -508,6 +572,26 @@ public static ConnectionMode parse(String value, String defaultValue) {
                     + "'change_streams' to capture changes via MongoDB Change Streams, update events do not contain full documents; "
                     + "'change_streams_update_full' (the default) to capture changes via MongoDB Change Streams, update events contain full documents");
 
+    public static final Field CAPTURE_SCOPE = Field.create("capture.scope")
+            .withDisplayName("Capture scope")
+            .withEnum(CaptureScope.class, CaptureScope.DEPLOYMENT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 2))
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.MEDIUM)
+            .withDescription("The scope of captured changes. "
+                    + "Options include: "
+                    + "'deployment' to capture changes from the entire MongoDB deployment; "
+                    + "'database' (the default) to capture changes from a specific MongoDB database");
```

Comment:
I think `deployment` is the default

Reply:
I don't think this will be needed elsewhere at the moment. But definitively I will keep that in mind.